config/api.yml

```diff
@@ -473,6 +473,29 @@
       :get:
       - :name: read
         :identifier: miq_cloud_networks_view
+  :cloud_volumes:
+    :description: Cloud Volumes
+    :identifier: cloud_volume
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
The current use case that I know needs it only as a primary collection. In that case, should I remove `- :subcollection`?